### PR TITLE
Swag @Routes update to remove /api in favour of basepath

### PR DESCRIPTION
**Pull request by radiosilence**
* `/api` should not be part of `@Router` because it is in the `basePath` now.
* 

Without this PR the swagger is incorrect and the client code generated from it will be broken.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -83,7 +83,7 @@ reviewable: swag ## Ensure a PR is ready for review.
 	@go mod tidy
 
 .PHONY: check-diff
-check-diff: reviewable ## Ensure branch is clean.
+check-diff: reviewable swag ## Ensure branch is clean.
```
Comment (gusfcarvalho) [inline Makefile:86]: reviewable already uses swag

File: Makefile [hunk 2]
```diff
@@ -99,4 +99,3 @@ debug.stop: ## Run docker-compose with debug
 
 swag:
 	@swag init
-	@swag fmt
```
Comment (gusfcarvalho) [inline Makefile:102]: if we don't format, it won't update files. would make `make swag` useless, no?

File: Makefile (hunk 1 above)
Comment (radiosilence) [reply Makefile:86]: Oh, fair

File: Makefile (hunk 2 above)
Comment (radiosilence) [reply Makefile:102]: Nah it was causing issues. Fmt just created weird broken diffs

Comment (radiosilence) [reply Makefile]: swag fmt only formats your go code

Comment (radiosilence) [reply Makefile]: >    (optional) Use swag fmt format the SWAG comment. (Please upgrade to the latest version)

Comment (radiosilence) [reply Makefile]: I think we had an edge case with the formatter where it wasn't formatting correctly and adding a tab each time for some reason

**Review by dominicmeddick — APPROVED** [review]

**Pull request merged**